integrationTest.sh

```diff
@@ -3,14 +3,17 @@ set -euo pipefail
 IFS=$'\n\t'
 
 # Unit tests first
-go test ./... 2>&1 | grep -v 'no test files'
+set +u
+if [ -z "$CIRCLECI" ]; then
+  go test ./... 2>&1 | grep -v 'no test files'
```

Comment:
Break the `go test` command out into a unit test workflow so it can run in their existing go docker image (rather than having to install go on this machine in order to run it in the same image as used for the integration tests).